model/Core/Properties/relationshipType.md

```diff
@@ -4,11 +4,13 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+Provides a reasonable estimation of the type of relation
+between two identified Elements from a developer perspective.
```

Comment:
I think the above 2 lines can be copied from the [similar section in the 2.3 spec](https://github.com/spdx/spdx-spec/blob/development/v2.3.1/chapters/relationships-between-SPDX-elements.md#1111-description):

`This field provides information about the relationship between two SPDX elements. For example, you can represent a relationship between two different Files, between a Package and a File, between two Packages, or between one SPDXDocument and another SPDXDocument.`

Reply:
Actually, this would be more appropriate for the description.  For the summary, it could be shorter: `Information about the relationship between two SPDX elements.`

Reply:
done